lib-es5/uploader.js

```diff
@@ -219,6 +219,29 @@ exports.create_zip = function create_zip(callback) {
   return exports.create_archive(callback, options, "zip");
 };
 
+exports.create_slideshow = function create_slideshow(options, callback) {
+  options.resource_type = "video";
```

Comment:
It would be more future-proof if you set it as a default value.
like:
`ensureOption(options, "resource_type", "video");`

Reply:
Fixed!